.github/workflows/lint.yml

```diff
@@ -12,9 +12,20 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.20"
+          go-version-file: 'go.mod'
```

Comment:
nice change. I'm glad we've been slowly updating all our connectors with some of these changes as we go through them.